web/src/search_suggestion.ts

```diff
@@ -184,6 +184,29 @@ function get_channel_suggestions(last: NarrowTerm, terms: NarrowTerm[]): Suggest
     });
 }
 
+function get_archived_channel_filter_suggestions(
```

Comment:
I would either include this with `get_channels_filter_suggestions` or rename that to `get_public_channels_filter_suggestions`

Reply:
Oh seeing [this conversation](https://github.com/zulip/zulip/pull/34799#discussion_r2127846327) now. I think ideally you could get them all into `get_channels_filter_suggestions`, but I'm not sure how difficult that would be.